### ISSUE TITLE
epoxy_is_desktop_gl does not detect GLES

### ISSUE THREAD

**Issue by Wallbraker** [anholt/libepoxy#147]
Tried to change a application using libepoxy use GLES instead of Desktop GL. And epoxy_is_desktop_gl continues to return true.

```bash
$ apt show libepoxy0 
Version: 1.3.1-1ubuntu1.17.04.2
```

```c
EGLint curapi;
eglQueryContext(eglGetCurrentDisplay(), eglGetCurrentContext(), EGL_CONTEXT_CLIENT_TYPE, &curapi);

fprintf(stderr, "ver: \"%s\"\n", glGetString(GL_VERSION));
fprintf(stderr, "epoxy_is_desktop_gl: \"%i\"\n", epoxy_is_desktop_gl());
fprintf(stderr, "EGL says: egl %i, gl %i\n", curapi == EGL_OPENGL_ES_API, curapi == EGL_OPENGL_API);
```

```
ver: "OpenGL ES 3.2 Mesa 17.0.7"
epoxy_is_desktop_gl: "1"
EGL says: egl 1, gl 0
```

**Comment by nwnk** [comment]
> ```Version: 1.3.1-1ubuntu1.17.04.2```

There have been quite a few fixes since then, do you see this issue with 1.4.3 or git master? I think at minimum 7c4817f2eed2faf0353c1ceb5f38b500f6aff7cf would make some difference here.

**Comment by Wallbraker** [comment]
Yes, upgrading to git master fixed my issue, thanks.
